cmd/client-s3.go

```diff
@@ -762,6 +762,10 @@ func (c *s3Client) MakeBucket(region string) *probe.Error {
 	}
 	e := c.api.MakeBucket(bucket, region)
 	if e != nil {
+		// Ignore bucket already existing error when ignoreExisting flag is enabled
+		if ignoreExisting && minio.ToErrorResponse(e).Code == "BucketAlreadyOwnedByYou" {
+			return nil
```

Comment:
There is one more Code to look for. `BucketAlreadyExists`  its reported by some regions.